### PR TITLE
feat: bump to v0.11.0-rc6 node. Introduction to da full/light nodes

### PR DESCRIPTION
**Pull request by Bidon15**
## Overview

- Bumping to node v0.11.0-rc6 for helm charts
- Fixing `if/else` statements in Chart.yaml after linter issues in `helm lint charts/node`
- Fixing logo uri for `charts/node` - pointing to a svg
- [x] dry-run is passing
- [x] templating is correct after refactoring 
- [x] config.toml files for full and light DA Nodes

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: charts/node/Chart.yaml [hunk 1]
```diff
@@ -15,10 +16,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: v0.11.0-rc3
+version: 0.11.0-rc6
```
Comment (Bidon15) [inline charts/node/Chart.yaml:19]: I removed the `v` to be compliant with the linter. It was noting that the `v` prefix is no bueno

File: charts/node/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: node
 description: A Helm chart for deploying celestia node on Kubernetes
+icon: https://company-223625.frontify.com/d/JoSwaZS4Mjpj/guidelines#/basics/logos/purple-watermark
```
Comment (Bidon15) [inline charts/node/Chart.yaml:4]: took from branding guidelines in https://company-223625.frontify.com/d/JoSwaZS4Mjpj/guidelines

File: charts/node/files/bridge-node/config.toml [hunk 3]
```diff
@@ -1,3 +1,7 @@
+[Node]
+  StartupTimeout = "2m0s"
+  ShutdownTimeout = "2m0s"
+
```
Comment (Bidon15) [inline charts/node/files/bridge-node/config.toml:4]: As you see the config toml changed again vs v0.11.0-rc3 
cc: @sysrex

File: charts/node/templates/sts-node.yaml [hunk 4]
```diff
@@ -87,7 +87,7 @@ spec:
             - name: {{ tpl .Values.celestiaNode.config.configmapName . }}
               mountPath: "{{ tpl .Values.celestiaNode.home.path . }}/{{ .Values.celestiaNode.config.file }}"
               subPath: {{ .Values.celestiaNode.config.file }}
-            {{- if .Values.celestiaNode.bootstrapper.true}}
+            {{- if eq .Values.celestiaNode.bootstrapper true -}}
```
Comment (Bidon15) [inline charts/node/templates/sts-node.yaml:90]: having the former if statement is violating linter rules. If we had the enabled: false in .yaml, then if statement could be good to pass

File: charts/node/Chart.yaml (hunk 1 above)
Comment (jrmanes) [reply charts/node/Chart.yaml:19]: do you mean the yaml lint?, maybe it's worth it to try with:
```yaml 
version: "v0.11.0-rc6"
```

or add the comment to skip this line, it's possible that we'll refer to this value from the templates and the tag without `v` doesn't exist :S

File: charts/node/files/bridge-node/config.toml [hunk 5]
```diff
@@ -14,10 +18,10 @@
   MutualPeers = []
   PeerExchange = true
   RoutingTableRefreshPeriod = "1m0s"
-[P2P.ConnManager]
-  Low = 800
-  High = 1000
-  GracePeriod = "1m0s"
+  [P2P.ConnManager]
```
Comment (jrmanes) [inline charts/node/files/bridge-node/config.toml:21]: and I thought that yaml files were hard to review 🥲 hahha

File: charts/node/values.yaml [hunk 6]
```diff
@@ -61,7 +61,7 @@ celestiaNode:
     registry: ghcr.io
     repository: celestiaorg
     name: celestia-node
-    tag: v0.11.0-rc6  #tag for v0.11.0-rc3
+    tag: v0.11.0-rc6  #tag for v0.11.0-rc6
```
Comment (jrmanes) [inline charts/node/values.yaml:64]: we could get rid of this comment, wdyt?

Comment (Bidon15) [reply charts/node/values.yaml:64]: Agreed

File: charts/node/Chart.yaml (hunk 1 above)
Comment (Bidon15) [reply charts/node/Chart.yaml:19]: IIRC, it's not idiomatic to put a v prefix in versioning of a chart itself. While the application version can contain a v prefix

Thus adding a version: "v0.11.0-rc6" will violate the common practice. 
Meaning that we will have something of node:0.11.0-rc6 for charts 

Do we still want to have a `v` and break expectations for helm users? 

cc: @sysrex

Comment (jrmanes) [reply charts/node/Chart.yaml:19]: you've convinced me 

also, I've checked some charts and yeah, they are not using the `v` prefix for the version

- [alertmanager](https://github.com/prometheus-community/helm-charts/blob/main/charts/alertmanager/Chart.yaml#L9-L10)
- [black-box](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-blackbox-exporter/Chart.yaml#L4-L5)

**Review by sysrex — APPROVED** [review]
LGTM

**Review by jrmanes — APPROVED** [review]
LGTM

**Pull request merged**